CP-CPS.md

```diff
@@ -332,7 +332,7 @@ See Section 2.2 of this document for Root and Subordinate CA certificate publica
 
 All Subscriber Certificates are made available to Subscribers via the ACME protocol.
 
-For each Subscriber Certificate issuance, ISRG signs a Precertificate and submits it to a selection of Certificate Transparency logs. Upon successful submission, ISRG attempts to issue a certificate that matches the Precertificate (per RFC 6962 Section 3.1) and embeds at least two of the resulting Signed Certificate Timestamps (SCTs). ISRG submits the resulting final certificate to a selection of Certificate Transparency logs on a best-effort basis.
+Prior to each Subscriber Certificate issuance, ISRG signs a corresponding Precertificate and submits it to a selection of Certificate Transparency logs. After each Subscriber Certificate issuance, ISRG submits the resulting certificate to a selection of Certificate Transparency logs on a best-effort basis.
```

Comment:
I think we should delete this section. It's enough to say that we publish certificates to the subscriber via ACME. SCT enforcement is done as a technical measure at validation time, by browsers. There's no root program or BR requirement for SCT inclusion.

Reply:
I've reduced what we say here to just "submit to CT on a best effort basis".